django_mongodb/expressions.py

```diff
@@ -81,6 +87,8 @@ def order_by(self, compiler, connection):
 def query(self, compiler, connection):  # noqa: ARG001
     raise NotSupportedError("Using a QuerySet in annotate() is not supported on MongoDB.")
 
+def raw_sql(self, compiler, connection):  # noqa: ARG001
+    raise NotSupportedError("QuerySet extras are not supported on MongoDB.")
```

Comment:
RawSQL can be used in places besides `QuerySet.extra()` so let's just say: "RawSQL is not supported..."